File: Ad-Hoc/__3000__/3128/3128.py
idade_1 = int(input())
idade_2 = int(input())

condicao = (idade_1 >= 14 and idade_2 >= 14) or \
           (idade_1 >= 18 or idade_2 >= 18)

print("YES" if (idade_1 >= 6 and idade_2 >= 6 and condicao) else "NO")

File: Ad-Hoc/__3000__/3128/3128.cpp
#include <iostream>
using namespace std;

int main(){
    int i1, i2;
    cin >> i1 >> i2;

    bool condicao = ((i1 >= 14 && i2 >= 14) || (i1 >= 18 || i2 >= 18)) && i1 >= 6 && i2 >= 6;
    cout << (condicao ? "YES" : "NO") << endl;

    return 0;
}